docs/configuration.md

```diff
@@ -431,25 +407,20 @@ And, conversely, the following would only format Jupyter Notebook files and not
 === "pyproject.toml"
 
     ```toml
-    [tool.ruff]
-    extend-include = ["*.ipynb"]
-
     [tool.ruff.lint]
     exclude = ["*.ipynb"]
     ```
 
 === "ruff.toml"
 
     ```toml
-    extend-include = ["*.ipynb"]
-
     [lint]
     exclude = ["*.ipynb"]
     ```
 
-Alternatively, pass the notebook file(s) to `ruff` on the command-line directly. For example,
-`ruff check /path/to/notebook.ipynb` will always lint `notebook.ipynb`. Similarly,
-`ruff format /path/to/notebook.ipynb` will always format `notebook.ipynb`.
+Alternatively, passing the notebook file(s) to `ruff` on the command-line will always lint or format
+the notebook. For example, `ruff check /path/to/notebook.ipynb` will always lint `notebook.ipynb`.
+Similarly, `ruff format /path/to/notebook.ipynb` will always format `notebook.ipynb`.
```

Comment:
I think we won't need this section anymore. we can move this section to another place that applies for all file types not just .ipynb